app/components/createContainer/createContainerController.js

```diff
@@ -478,6 +485,7 @@ function ($q, $scope, $state, $timeout, $transition$, $filter, Container, Contai
       loadFromContainerPortBindings(d);
       loadFromContainerVolumes(d);
       loadFromContainerNetworkConfig(d);
+      loadFromContainerMacAddress(d);
       loadFromContainerEnvrionmentVariables(d);
```

Comment:
Mind renaming this function to correct the typo ? `loadFromContainerEnvrionmentVariables` -> `loadFromContainerEnvironmentVariables`